File: Algorithm/Programmers/Level1/solution(29).py
# 2016년

def solution(a, b):
    answer = ''
    day_name = ["FRI","SAT","SUN","MON","TUE","WED","THU"]
    month_end = [31,29,31,30,31,30,31,31,30,31,30,31]
    month_sum = b
    for i in range(a-1):
        month_sum += month_end[i]
    answer = day_name[month_sum%7-1]
    return answer

File: Algorithm/Programmers/Level1/solution(29).java
// 2016년

class Solution {
    public String solution(int a, int b) {
        String answer = "";
        String[] day = {"THU", "FRI","SAT","SUN","MON","TUE","WED"};
        int[] month = {31,29,31,30,31,30,31,31,30,31,30,31};
        int sum = b;
        
        for (int i = 0; i < a-1; i++)
            sum += month[i];
        
        answer = day[sum%7];
        
        return answer;
    }
}